modules/social_features/social_activity/social_activity.install

```diff
@@ -167,3 +172,90 @@ function social_activity_update_8801() {
   $view->set('display', $displays);
   $view->save(TRUE);
 }
+
+/**
+ * Create "Featured" view mode/display for activity.
+ */
+function social_activity_update_8802() {
+  // Create a new activity featured entity view mode.
+  EntityViewMode::create([
+    'targetEntityType' => 'activity',
+    'id' => 'activity.featured',
+    'status' => TRUE,
+    'label' => t('Featured'),
+  ])->save();
+
+  // Create the corresponding entity view display for activity entity type.
+  $display = EntityViewDisplay::load('activity.activity.default')->toArray();
+  $display['content']['field_activity_entity']['settings']['node'] = [
+    'view_mode' => 'featured',
+  ];
+  $display = array_merge($display, [
+    'uuid' => NULL,
+    'targetEntityType' => 'activity',
+    'bundle' => 'activity',
+    'mode' => 'featured',
+  ]);
+  EntityViewDisplay::create($display)->save();
+}
+
+/**
+ * Create "Featured" view mode/display for post.
+ *
+ * @todo: should be moved into social_post module.
+ */
+function social_activity_update_8803() {
+  // Create a new post featured entity view mode.
+  EntityViewMode::create([
```

Comment:
@volodymyr-sydor-lemberg Please move this to social_post module.

Reply:
done